### PR TITLE
Correction of NRE, Paramatrized attenuation and Reduced the current attenuation of Jukebox

### LINKED ISSUE #4624: Create the jukebox prefab [First Iteration] TB
## Description

The bar needs a jukebox. We need the first iteration of the jukebox created and mapped to the bar of outpost station.

Criteria:

- Find the jukebox sprites in the repo: 
image
- Create a new interactable machine prefab (you can duplicate an existing machine and change it to make it into the jukebox prefab)
- [Speak to ReiderJack about this part if you need some quick directions] Find a way to play the music that plays on the lobby screen from the jukebox
- Make sure the jukebox music can only be heard up to a reasonable distance and follows the other sounds when it comes to being muffled when heard through walls (soundmanager should handle all of this you just need to know how the new soundmanager works as it was recently refactored by reider)
- Make sure the music playing is network synced properly by testing with 2 clients (you can mute one to make sure the other can hear)
- Create a small pop up window for the players that click the jukebox to skip songs or replay songs. Show the song title in this pop up window. Make sure the window closes when the player moves away or gets knocked over

### PR DESCRIPTION
**Pull request by JesterX666**
Correction of NRE, Paramatrized attenuation and Reduced the current attenuation of Jukebox.

I could have use a StringBuilder instead of string interpolation, but I find it "clearer" the way I did it.

NRE Detail:

NullReferenceException: Object reference not set to an instance of an object
Assets.Scripts.Messages.Server.SoundMessages.PlaySoundMessage.ToString () (at /github/workspace/UnityProject/Assets/Scripts/Messages/Server/SoundMessages/PlaySoundMessage.cs:154)
Assets.Scripts.Messages.Server.SoundMessages.PlaySoundMessage.Process () (at /github/workspace/UnityProject/Assets/Scripts/Messages/Server/SoundMessages/PlaySoundMessage.cs:28)
GameMessageBase.Process (Mirror.NetworkConnection sentBy) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:25)
GameMessageBase.PreProcess (Mirror.NetworkConnection sentBy, GameMessageBase b) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:18)
Mirror.MessagePacker+<>c__DisplayClass6_0`2[T,C].<MessageHandler>b__0 (Mirror.NetworkConnection conn, Mirror.NetworkReader reader, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/MessagePacker.cs:146)
Mirror.NetworkConnection.InvokeHandler (System.Int32 msgType, Mirror.NetworkReader reader, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:219)
Mirror.NetworkConnection.TransportReceive (System.ArraySegment`1[T] buffer, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:272)
Mirror.ULocalConnectionToServer.Update () (at /github/workspace/UnityProject/Assets/Mirror/Runtime/LocalConnections.cs:110)
Mirror.NetworkClient.Update () (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkClient.cs:259)
Mirror.NetworkManager.LateUpdate () (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkManager.cs:255)

### CODE REVIEW

**Comment by ThatDan123** [comment]
I changed the prefab in the mapping PR as the sprite child wasnt set to 0,0,0

File: UnityProject/Assets/Scripts/Machines/Jukebox.cs [hunk 1]
```diff
@@ -34,6 +34,12 @@ public class Jukebox : NetworkBehaviour, IAPCPowered
 	[SerializeField]
 	private SpriteDataSO SpriteDamaged;
 
+	[SerializeField]
+	private float MinSoundDistance;
+
+	[SerializeField]
+	private float MaxSoundDistance;
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Machines/Jukebox.cs]: Could you assign any non-public `[SerializeField]`s to some value, like `default` or `null`. Just helps reduce compiler warnings. Thanks.

Comment (JesterX666) [reply UnityProject/Assets/Scripts/Machines/Jukebox.cs]: Done

**Comment by Bod9001** [comment]
it is ready?

**Comment by corp-0** [comment]
we could merge test it

**Comment by corp-0** [comment]
now I feel attenuation is way too strong. You're 3 tiles away and can barely hear the music

**Comment by GenericDaemon** [comment]
Yeah, tested on staging and the music is a barely audible muffled mess unless one is standing right besides the jukebox

**Comment by corp-0** [comment]
It should be fixed now, sadly staging doesn't have the latest version yet because we're facing other problems with the build server.

**Pull request merged**